changelog.json

```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.5.1-beta5": [
+      "[Improved] Commit conflicted files warning - #6381",
+      "[Improved] Dismissable merge conflict dialog and associated banner - #6379 #6380",
+      "[Fixed] Fix feature flag for readme overwrite warning so that it shows on beta - #6412"
```

Comment:
:art: this won't be needed in the final production release, right?

Reply:
We can get to this as part of reviewing the release notes for production :shipit: